src/wp-includes/class-wp-theme.php

```diff
@@ -327,9 +339,10 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 			$this->cache_add(
```

Comment:
I think here we need to add the check for whether it's a block theme, or at least just set it to `false`.
```suggestion
			$this->block_theme = false;
			$this->cache_add(
```